src/config.py

```diff
@@ -282,5 +284,5 @@ def zookeeper_command(self) -> str:
         Returns:
             String of startup command and expected config filepath
         """
-        entrypoint = "/opt/kafka/bin/zookeeper-server-start.sh"
-        return f"{entrypoint} {self.properties_filepath}"
+        entrypoint = "/opt/zookeeper/bin/zkServer.sh"
+        return f"{entrypoint} --config {self.default_config_path} start-foreground"
```

Comment:
What was the difference between start and start-foreground? I'm sure it's fine I just don't remember

Reply:
`start` is a forking process which is not what you want for the container so I went with `start-foreground` to avoid the process forking.